fights/base_action.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+
+from .base_state import BaseState
+
+
+class BaseAction(ABC):
```

Comment:
Do we really need a class for an action?

A typical action will most likely be of type `numpy.typing.ArrayLike`.